CONTRIBUTING.md

```diff
@@ -163,6 +170,20 @@ For votes regarding non-coding issues such as procedure changes, the requirement
 that a veto is accompanied by a *technical* justification is relaxed somewhat,
 though a well reasoned justification must still be included.
 
+### Reversions
+
+A *-1* vote on any recently merged pull request requires an immediate
+reversion of the merged pull request. The backout of such a pull request
+invokes a mandatory, minimum 72 hour, review period.
+
+- Recently merged pull requests are defined as a being within the past 168 hours (7
+  days), so as to not prevent forward progress, while allowing for reversions of
+  things merged without proper review and consensus.
+- The person issuing the -1 vote will, upon commenting *-1* with technical
+  justification per the vetoes section, immediately open and merge a pull request to
+  revert the original merge in question outside of the normal requirement for a
+  +1 from "someone other than the author", per "Handling Pull Requests"
```

Comment:
I think given that a technical justification needs to be given - so I would suggest someone other than the -1 issuer needs to decide the justification is "technical" - not that they agree with it - just that the reason given is in fact technical.

"The person issuing the -1 vote will, upon commenting *-1* with technical justification per the vetoes section, immediately open a pull request to revert the original merge in question. If any committer other than the -1 issuer deems the justification technical - regardless of whether they agree with justification - that committer must then merge the pull request to revert."